.github/workflows/build.yml

```diff
@@ -15,11 +15,8 @@ jobs:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
 
-      - name: Build compliance-masonry
-        run: go build cmd/compliance-masonry/compliance-masonry.go
-
-      - name: Build masonry
-        run: go build cmd/masonry/masonry.go
+      - name: Build compliance-masonry and masonry
+        run: go build -o _output/ ./cmd/...
```

Comment:
@aegershman @isimluk merged the multiple binary step to this command. Thoughts?

Reply:
I think this is fine for now 👍 I think it's good enough to be merged in, and this could always be revised on specifics. If I understand correctly, I don't think the current output of the built artifact is used, I get the impression it's just being used to determine if the cli can be built at all.

Reply:
Yes, that's true. Okay. Thanks! Would you be willing to look at getting MacOS and Windows up and running?